packages/uhk-common/src/models/uhk-products.ts

```diff
@@ -45,3 +48,77 @@ export const UHK_DEVICES: Array<UhkDeviceProduct> = [
     UHK_60_DEVICE,
     UHK_60_V2_DEVICE
 ];
+
+export interface UhkModule {
+    type: UhkProductTypes.Module;
+    id: number;
+    name: string;
+    slotId: ModuleSlotToId;
+    slotName: string;
+    i2cAddress: ModuleSlotToI2cAddress;
+    firmwareUpgradeSupported: boolean;
+    bootloaderPingReconnectMsg: string;
+}
+
+export const LEFT_HALF_MODULE: UhkModule = {
+    type: UhkProductTypes.Module,
+    id: 1,
+    name: 'Left Keyboard Half',
+    slotId: ModuleSlotToId.leftHalf,
+    slotName: 'LeftKeyboardHalf',
+    i2cAddress: ModuleSlotToI2cAddress.leftHalf,
+    firmwareUpgradeSupported: true,
+    bootloaderPingReconnectMsg: 'Cannot ping the bootloader. Please remove the bridge cable, and keep reconnecting the left keyboard half until you see this message.'
+};
+
+export const LEFT_KEY_CLUSTER_MODULE: UhkModule = {
+    type: UhkProductTypes.Module,
+    id: 2,
+    name: 'Left Key Cluster',
+    slotId: ModuleSlotToId.leftModule,
+    slotName: 'KeyClusterLeft',
+    i2cAddress: ModuleSlotToI2cAddress.leftModule,
+    firmwareUpgradeSupported: true,
+    bootloaderPingReconnectMsg: 'Cannot ping the bootloader. Please remove the "Left Key Cluster" module, and keep reconnecting it until you see this message.'
```

Comment:
Is there an easy way to not repeat this for every module?

Reply:
I removed the `type` and `slotName` properties because we don't use it anymore.
The `firmwareUpgradeSupported` is a feature flag that helps to decide Agent supports the firmware update of the module or not e.g. touchpad currently
I would like to remove the `bootloaderPingReconnectMsg` property if we can figure it out an error message that meaningful and contains a place holder for the module name like: `Cannot ping the bootloader. Please remove the "${moduleName}" module, and keep reconnecting it until you see this message.`. I use this distinction because the  message of the left half is different `Cannot ping the bootloader. Please remove the bridge cable, and keep reconnecting the left keyboard half until you see this message.`

Reply:
I only meant to generalize `bootloaderPingReconnectMsg` for modules. The suggested string is good.